### PR TITLE
Add donation information via Liberapay

### PR DESCRIPTION
**Pull request by eht16**
Also remove the outdated PayPal logo and use a direct link to PayPal.

### CODE REVIEW

**Comment by eht16** [comment]
It looks like: 
geany_org_donate_liberapay

I also changed the redirect for "https://www.geany.org/service/donate/" (used in Geany in the Help menu) to not directly point to PayPal but instead to the donate page on our website, so users can see all available options.

File: page_content/about/donate.md [hunk 1]
```diff
@@ -15,23 +15,24 @@ More than financial support, the Geany project needs help with:
 
 If you want to donate money to the Geany project, it will be used to support further development of Geany, to pay running costs for hosting and domains and support presenting Geany on various [Open Source Conferences][5].
 
-Currently, there are two options to donate money: via bank transfer and Paypal.
-Bank transfer to the [Geany e.V.][8] account is the preferred way for donating money as the money is transferred directly without a third party involved and Paypal fees can be avoided.
+There are different options to donate, including via bank transfer and Liberapay.
```
Comment (dmaphy) [inline page_content/about/donate.md:18]:
```suggestion
There are different options to donate, including bank transfer and Liberapay.
```

File: page_content/about/donate.md [hunk 2]
```diff
@@ -15,23 +15,24 @@ More than financial support, the Geany project needs help with:
...
+Bank transfer to the [Geany e.V.][8] account is the preferred way for donating money as the money is transferred directly without a third party involved and Liberapay/Paypal fees can be avoided.
```
Comment (dmaphy) [inline page_content/about/donate.md:19]:
```suggestion
Bank transfer to the [Geany e.V.][8] account is the preferred way for donating money as the money is transferred directly without a third party involved and additional fees can be avoided.
```

**Review by dmaphy — CHANGES_REQUESTED** [review]

**Comment by eht16** [comment]
Thanks for the feedback, completely incoporated.
I will squash the commits before merge.

**Pull request merged**